src/endpoints/accounts.js

```diff
@@ -0,0 +1,52 @@
+import BaseExtend from '../extends/base'
+
+class AccountsEndpoint extends BaseExtend {
+  constructor(endpoint) {
+    super(endpoint)
+
+    this.endpoint = 'accounts'
+    this.version = 'v1'
+  }
+
+  Stores() {
+    return this.request.send(
+      `${this.endpoint}/stores`,
+      'GET',
+      undefined,
+      undefined,
+      this
+    )
+  }
+
+  Store(id) {
+    return this.request.send(
+      `${this.endpoint}/stores/${id}`,
+      'GET',
+      undefined,
+      undefined,
+      this
+    )
+  }
+
+  StoresSwitch(storeId) {
```

Comment:
`SwitchStore` may be a clearer name?

Reply:
Yes, I needed that.
Thanks, Sam
Fixing